netbout-web/src/main/java/com/netbout/dynamo/DyAlias.java

```diff
@@ -113,7 +113,9 @@ public String email() throws IOException {
 
     @Override
     public void email(final String email) throws IOException {
-        if (!email.matches("[a-z_\\.@\\-A-Z0-9]+")) {
+        final String pattern = "[a-z_\\.@\\-A-Z0-9]";
```

Comment:
@bdragan performance-wise I think it's better to compile the regex and keep it in a constant...